HW3/YuliiaKutsyk/task_3_1.py

```diff
@@ -18,14 +17,6 @@
 If the implementation is hard to explain, it's a bad idea.
 If the implementation is easy to explain, it may be a good idea.
 Namespaces are one honking great idea -- let's do more of those!"""
-
-number_better = zen.find("better")
-print("Кількість слів 'better':",number_better)
-number_never = zen.find("never")
-print("Кількість слів 'never':",number_never)
-number_is = zen.find("is")
-print("Кількість слів 'is':",number_is)
-
-print(f"Ось весь текст в верхньому регістрі: \n{zen.upper()}")
-
-print(f"Ось весь текст з заміною 'i' на '&': \n{zen.replace('i','&')}")
\ No newline at end of file
+print('better counts ', zen.find('better'))
+print('never counts ', zen.find('never'))
```

Comment:
must be
print('never counts ', zen.count('never'))
print('is counts ', zen.count('is'))